Mathlib/Data/String/Basic.lean

```diff
@@ -159,26 +137,20 @@ theorem toList_singleton (c : Char) : (String.singleton c).toList = [c] :=
   rfl
 #align string.to_list_singleton String.toList_singleton
 
-theorem toList_nonempty : ∀ {s : String}, s ≠ "" → s.toList = s.head :: (s.popn 1).toList
```

Comment:
`String.popn` is a duplicate of `String.drop`. See https://github.com/leanprover-community/mathlib4/pull/3789.